app/assets/javascripts/users/settings/teams/invite_users_modal.js

```diff
@@ -105,6 +105,10 @@
           inviteBtn.show();
           inviteWithRoleDiv.hide();
           break;
+        case 'invite_to_team_with_role_selector':
```

Comment:
Don't add a new type, just use invite_to_team_with_role

Reply:
Thanks for the feedback